src/eth_rpc/config.rs

```diff
@@ -14,4 +14,14 @@ impl RPCConfig {
             .map_err(|_| eyre!("Missing mandatory environment variable: KAKAROT_RPC_URL"))?;
         Ok(Self::new(socket_addr))
     }
+
+    pub fn from_port(port: u16) -> Result<Self> {
```

Comment:
thanks a lot, we've been needing this for a long time, was going to impl. it for debug_rawBlock 

if we wanted to go one step further we could decouple testing environment from actual .env file, it feels wrong that our tests rely on non-committed .env files, 
But since it probably belongs in a smol non-related PR, could you create an issue?

I suggest adding to the Config struct a new method `::new_test_config()` method, or something similar, under `testing` feature. Let me know if you have a better idea 👑

Reply:
Good idea, just opened a PR here: https://github.com/kkrt-labs/kakarot-rpc/issues/879